templates/keepalived.conf.j2

```diff
@@ -204,4 +207,4 @@ virtual_server group {{ vserver_group.name }} {
   {% endfor %}
 }
 {% endfor %}
-{% endif %}
\ No newline at end of file
+{% endif %}
```

Comment:
can you remove this change?

While this is technically okay (and better) to have an extra newline, doing so will trigger (for our existing users) a restart of keepalived (because the template has changed) on the next release, for no real good reason.

Reply:
I will have a look. Seems my Editor automatically appends this.